### PR TITLE
N17: Release

### PR DESCRIPTION
**Pull request by ozkriff**
Preview (https://github.com/rust-gamedev/rust-gamedev.github.io/commit/2fe5c722a1fa37750a91745c3db1e7c1697ee8ca): https://rust-gamedev-preview.netlify.app/posts/newsletter-017

Everyone is invited to review!

:timer_clock: Hope to merge this in an hour or so if there're no objections.

_Part of #408 ("Newsletter 17")_

### CODE REVIEW

File: content/posts/newsletter-017/index.md [hunk 1]
```diff
@@ -61,6 +57,32 @@ _Discussions:
 If needed, a section can be split into subsections with a "------" delimiter.
 -->
 
+## [Rust GameDev Podcast #4][podcast-4]
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-017/index.md:60]: Tangentally related - we should call out the next meetup in the next newsletter :)

**Review by AlexEne — APPROVED** [review]

**Review by 17cupsofcoffee — APPROVED** [review]
🚀 🚀 🚀 🚀

File: content/posts/newsletter-017/index.md (hunk 1 above)
Comment (AngelOnFira) [reply content/posts/newsletter-017/index.md:60]: @ozkriff I'll quickly do a writeup that you can put wherever if you like

Comment (AngelOnFira) [reply content/posts/newsletter-017/index.md:60]:
```
The Rust Gamedev meetup will happen on the second Saturday of every month at 16:00 GMT+0. The meetup will be an opportunity to chat with others from the Rust game development community and see what people are working on. The meetup will take place on the Rust Gamedev Discord server:

https://discord.gg/yNtPTb2
```

meetup

**Review by AngelOnFira — APPROVED** [review]
Looks great :100:

**Comment by cart** [comment]
Github won't let me point out the relevant section, but the Bevy section was changed to use semicolons instead of periods or commas to end bulleted lists, which feels a bit awkward nothing else in the newsletter does that:

```
- a WebGL2 render backend;
- cross platform main function: `#[bevy_main]`;
- live shader reloading;
- Schedule v2 with custom stages, run criteria, fixed timestep,
  and typed stage builders;
- States that allow you to enable/disable systems
  according to the state your app is in;
- scenes can now be spawned as children;
- dynamic linking that significantly cuts compile times.
- a new general-purpose Rust reflection API;
- 3D texture assets;
- HIDPI fixes;
- GLTF, timer, text layout, task system, logging & profiling improvements;
- ECS improvements: flexible system parameters, improved query filter API,
  and system inputs/outputs/chaining;
- Apple Silicon support!
```

File: content/posts/newsletter-017/index.md (hunk 1 above)
Comment (ozkriff) [reply content/posts/newsletter-017/index.md:77]: added

**Comment by ozkriff** [comment]
:timer_clock: merging in 20 minutes

**Comment by kvark** [comment]
> A plugin integrating Bevy. ECS with doryen-rs Ascii roguelike library.

Was meant to be "Bevy's ECS" ?

**Review by kvark — APPROVED** [review]
Glad to see this is happening, thank you!

**Comment by cart** [comment]
> Was meant to be "Bevy's ECS" ?

Yup. Looks like that got added in this style-fix commit:
https://github.com/rust-gamedev/rust-gamedev.github.io/pull/436/commits/5cc9150c8bdb464f0524724079175df2c13c9507

**Comment by ozkriff** [comment]
Ok, so I'm merging this, but feel free to report other issues if you happen to see them - I'll fix them in separate commits.

**Comment by AngelOnFira** [comment]
:tada:

**Pull request merged**